ios/FluentUI/Card/CardView.swift

```diff
@@ -0,0 +1,503 @@
+                setupLayoutConstraints()
+            }
+        }
+    }
+    @objc open var secondaryText: String? {
+        didSet {
+            if secondaryText != oldValue {
+                secondaryLabel.text = secondaryText
+                setupLayoutConstraints()
+            }
+        }
+    }
+    @objc open var icon: UIImage? {
+        didSet {
+            if icon != oldValue {
+                if let iconView = iconView {
+                    iconView.image = icon
+                }
+            }
+        }
+    }
+    @objc open var backgroundStyle: CardBackgroundStyle = .neutral {
+        didSet {
+            if backgroundStyle != oldValue {
+                setupBackgroundColor()
+            }
+        }
+    }
+    @objc open var titleNumberOfLines: Int = Constants.titleDefaultNumberOfLines {
+        didSet {
+            if titleNumberOfLines != oldValue {
+                primaryLabel.numberOfLines = titleNumberOfLines
+                setupLayoutConstraints()
+            }
+        }
+    }
+    @objc open var subtitleNumberOfLines: Int = Constants.subtitleDefaultNumberOfLines {
+        didSet {
+            if subtitleNumberOfLines != oldValue {
+                secondaryLabel.numberOfLines = subtitleNumberOfLines
+                setupLayoutConstraints()
+            }
+        }
+    }
+    @objc open var cardBackgroundColor: UIColor = Constants.defualtBackgroundColor {
+        didSet {
+            if cardBackgroundColor != oldValue {
+                setupBackgroundColor()
+            }
+        }
+    }
+
+    /**
+     Initializes `CardView`
+     
+     - Parameter style: The style of the card
+     - Parameter title: The title of the card
+     - Parameter subtitle: The subtitle of the card - optional
+     - Parameter icon: The icon of the card - optional
+     - Parameter image: The featured image of the card (small, medium, or announcement) - optional
+     - Parameter backgroundStyle: The Card's background style; appColor, neutral, or custom.
+     - Parameter buttonTitle: The button's title (announcement type only) - optional
+     **/
+    @objc public init(style: CardStyle, title: String, subtitle: String? = nil, icon: UIImage? = nil, image: UIImage? = nil, backgroundStyle: CardBackgroundStyle, buttonTitle: String? = nil) {
+        self.primaryText = title
+        self.style = style
+        self.secondaryText = subtitle
+        self.icon = icon
+        self.image = image
+        self.buttonTitle = buttonTitle
+        self.backgroundStyle = backgroundStyle
+
+        super.init(frame: .zero)
+
+        configureSubviews()
+    }
+
+    @available(*, unavailable)
+    @objc public required init?(coder: NSCoder) {
+        preconditionFailure("init(coder:) has not been implemented")
+    }
+
+    private func configureSubviews() {
+        translatesAutoresizingMaskIntoConstraints = false
+
+        // view border and background
+        layer.borderColor = Constants.borderColor.cgColor
```

Comment:
Does the border color need to change between light and dark mode? That won't happen when CGColors are used.

Reply:
There is no border for dark mode according to Yakun and Kaan. What would you recommend I use for this case?
I currently have a UIColor with the required border color in light mode and .clear color for dark mode. But yeah they don't get updated when I switch between dark/light modes.

Reply:
You would need to update the layer's border color when switching between light and dark mode.